openapi/components/schemas/Membership.yaml

```
@@ -26,10 +26,10 @@ properties:
     type: object
     properties:
       id:
-        description: Organization ID.
         type: string
-        allOf:
-          - $ref: ./ResourceId.yaml
+        description: Organization ID.
+        maxLength: 50
+        example: user_0YVCEFR9PRDMP9RAB31MHNPDHF
```

```suggestion
        example: usr_0YVCEFR9PRDMP9RAB31MHNPDHF
```
Also, it is not clear should it be user or organization.

---

True, updated, thanks